src/components/home/devRevenueCosts/Options.js

```diff
@@ -0,0 +1,74 @@
+import { RevenueCostsA, RevenueCostsM } from './Data';
+
+export const RevenueCostsActual = {
+  data: RevenueCostsA,
+  isGroup: true,
+  seriesField: 'name',
+  xField: 'projectName',
+  yField: 'value',
+  yAxis: {
+    min: 0,
+    max: 600000,
+  },
+  minColumnWidth: 20,
+  maxColumnWidth: 20,
+  dodgePadding: 6,
+  color: ({ name }) => {
+    const colorMap = {
+      'Grand Total Costs': '#FF9F5A',
+    };
+    return colorMap[name] || '#7BB99F';
+  },
+  legend: {
+    position: 'top-right',
+    layout: 'horizontal',
+    marker: {
+      symbol: 'circle',
+      r: 5,
+      lineWidth: 2,
+      strokeColor: ({ color }) => color, 
+      fill: 'transparent',
+    },
+    },
+};
+
+export const RevenueCostsMonth = {
+  data: RevenueCostsM.flat(),
+  isGroup: true,
+  seriesField: 'type',
+  xField: 'type', 
+  yField: 'value',
+  groupField: 'name',
+  yAxis: {
+    min: 0,
+    max: 260000,
+  },
+  minColumnWidth: 20,
+  maxColumnWidth: 20,
+  dodgePadding: 6,
+  color: ({ type }) => {
+    const colorMap = {
```

Comment:
this is good for demo purpose but we will expect live data and there will not be Grand Total Planned Revenue project and than this colors will not work. You need to figure out some different approach to do this. Myb you can create array of 10 colors and than randomly pick color for each project and than set al aver places.

Reply:
To do that you will need to upadate this RevenueCostsMonth to be function that will take parameter color.